EFTools.msbuild

```diff
@@ -90,20 +93,35 @@
     </Target>
 
     <Target Name="BuildTestHost">
-        <MSBuild Targets="Build"
+        <Message Condition="'$(IsDev14OrLower)' != 'True'"
```

Comment:
Would it be possible to avoid the repetition by adding the message in a common target that all these depend on?